TG-Netflix/src/components/account-page/ProfileSection.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from "react";
+import { AccountSection } from "../../pages/account/AccountPage.styled";
+import { data } from "../../pages/account/account";
+import { BsChevronDown, BsChevronUp } from "react-icons/bs";
+
+export default function ProfileSection() {
+  const [collapse, setCollapse] = useState([]);
+
+  // Zorgt ervoor dat enkel het profiel waar op wordt geklikt wordt opengeklapt en gesloten
+  function handleClick(param) {
+    if (!collapse.includes(param)) {
+      setCollapse([...collapse, param]);
+    } else {
+      let itemToRemove = collapse.find((item) => item === param);
+      if (itemToRemove) {
+        setCollapse(collapse.filter((item, index) => item !== itemToRemove));
+      }
+    }
+  }
+
+  return (
+    <AccountSection>
+      <div className="account-section-header">
+        <h2>PROFILE & PARENTAL CONTROLS</h2>
+      </div>
+      <div className="subsections-container">
+        {data.profiles.map((profile) => (
+          <div className="profile-container" key={profile.id}>
+            <div
+              className="profile-section"
+              onClick={() => handleClick(`${profile.id}`)}
+            >
+              <div className="profile-section-person">
+                <img src={profile.avatar} />
+                <div className="name-rating">
+                  <p>{profile.name}</p>
+                  <p>{profile.rating}</p>
+                </div>
+              </div>
+              {collapse === `${profile.id}` ? (
+                <BsChevronUp />
+              ) : (
+                <BsChevronDown />
+              )}
+            </div>
+            {profile.id + 1 <= Object.keys(data.profiles).length ? (
+              <hr className="profile-divider" />
+            ) : null}
+            {collapse.includes(`${profile.id}`) ? (
+              <>
+                <div className="profile-subsection">
+                  <div>
+                    <p>Language</p>
+                    <p>{profile.language}</p>
```

Comment:
de 'instellingen' zijn grijs ipv zwart, dit klopt wel bij "John / All maturity ratings" (John = zwart, All maturity ratings is grijs), maar uitgeklapt hebben de items allemaal tekstkleur zwart